apps/src/templates/instructions/TopInstructions.jsx

```diff
@@ -66,6 +66,16 @@ var TopInstructions = React.createClass({
     collapsed: React.PropTypes.bool.isRequired,
     onToggleCollapsed: React.PropTypes.func.isRequired,
     onChangeHeight: React.PropTypes.func.isRequired,
+    onLoadImage: React.PropTypes.func.isRequired
+  },
+
+  /**
+   * Called externally
+   * @returns {number} The height of the rendered contents in pixels
+   */
+  getContentHeight: function () {
+    var instructionsContent = this.refs.instructions.refs.instructionsMarkdown;
+    return $(ReactDOM.findDOMNode(instructionsContent)).outerHeight(true);
```

Comment:
Still looking at a child's refs here, but I could go either way on providing another helper method.  Seems like `TopInstructions` knows a lot about what its contents look like, and we're fine with that.